QMLComponents/components/JASP/Controls/ColorPalette.qml

```diff
@@ -27,15 +27,17 @@ DropDown
 	indexDefaultValue: 0
 	values:
 	[
-		{ label: qsTr("Colorblind"),		value: "colorblind"		},
-		{ label: qsTr("Colorblind #2"),		value: "colorblind2"	},
-		{ label: qsTr("Colorblind #3"),		value: "colorblind3"	},
-		{ label: qsTr("JASP"),				value: "jaspPalette"			},
-		{ label: qsTr("Viridis"),			value: "viridis"		},
-		{ label: qsTr("ggplot2"),			value: "ggplot2"		},
-		{ label: qsTr("Gray"),				value: "gray"			},
-		{ label: qsTr("Blue"),				value: "blue"			},
-		{ label: qsTr("Sports teams: NBA"),	value: "sportsTeamsNBA"	},
-		{ label: qsTr("Grand Budapest"),	value: "grandBudapest"	}
+		{ label: qsTr("Colorblind #3"),			   value: "colorblind3"	   },
+		{ label: qsTr("Colorblind #4"),			   value: "colorblind4"	   },
+		{ label: qsTr("Viridis"),				   value: "viridis"		   },
+		{ label: qsTr("Inferno"),				   value: "inferno"		   },
+		{ label: qsTr("JASP"),					   value: "jaspPalette"	   },
```

Comment:
I'm not sure if the naming makes sense here. Why not do Colorblind 1 through n and we mention elsewhere that this was different before version XXX? Also, please consistently use tabs for the indents instead of spaces.